appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -110,7 +110,7 @@
 	{
 		UserDto user = messageService.prepareMessageCollection(userService.getPrivateValidatedUser(userId));
 		Page<MessageDto> messages = messageService.getReceivedMessages(user, onlyUnreadMessages, pageable);
-		return createOkResponse(pagedResourcesAssembler, user, messages);
+		return createOkResponse(user, messages, pagedResourcesAssembler);
```

Comment:
Wouldn't it be more natural to have the messages as first parameter?

Reply:
Not to me. It's a feeling, I think, and I feel the user should go first and then the messages for that user.